### PR TITLE
updating readme

### PR DESCRIPTION
**Pull request by AliceWonderland**
First edit for README file. For grammar and punctuation.

Fixes https://github.com/openebs/openebs/issues/501

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,25 +57,25 @@ host#2$ mkdir /mnt/store2
 .
 ### Launch Controller
 
-Create a controller on host#1, that will create *demo-vol1* with capacity of 10G. 
+Create a Controller on host#1, that will create *demo-vol1* with capacity of 10G.
 
 ```
-host#1$ sudo docker run -d --network="host" -P --expose 3260 --expose 9501 --name ctrl openebs/jiva launch controller --frontend gotgt --frontendIP 172.18.200.101 demo-vol1 10G
+host#1$ sudo docker run -d --network="host" -P --expose 3260 --expose 9501 --name ctrl openebs/jiva launch Controller --frontend gotgt --frontendIP 172.18.200.101 demo-vol1 10G
```
Comment (vharsh) [inline README.md]: Change it to `controller`

File: README.md [hunk 2]
```diff
@@ -55,25 +57,25 @@ host#2$ mkdir /mnt/store2
...
-  
+
 .
 ### Launch Replica
 
 Create the Replica on host#1, for demo-vol1 with data stored in /mnt/store1
 
 ```
-docker run -d --network="host" -P --expose 9502-9504 --expose 9700-9800 -v /mnt/stor1:/stor1 --name replica-1 <openebs/jiva> launch replica --frontendIP 172.18.200.101 --listen 172.18.200.102:9502 --size 10G /stor1
+docker run -d --network="host" -P --expose 9502-9504 --expose 9700-9800 -v /mnt/stor1:/stor1 --name Replica-1 <openebs/jiva> launch Replica --frontendIP 172.18.200.101 --listen 172.18.200.102:9502 --size 10G /stor1
```
Comment (vharsh) [inline README.md]: Change it to `replica`

File: README.md [hunk 3]
```diff
@@ -55,25 +57,25 @@ host#2$ mkdir /mnt/store2
...
 ```
```
Comment (kmova) [inline README.md]: "launch Replica" -> "launch replica"

File: README.md [hunk 4]
```diff
@@ -55,25 +57,25 @@ host#2$ mkdir /mnt/store2
...
-.   
+.
 Create the backend container on host#2, for demo-vol1 with data stored in /mnt/store1
 
 ```
-docker run -d --network="host" -P --expose 9502-9504 --expose 9700-9800 -v /mnt/stor2:/stor2 --name replica-2 <openebs/jiva> launch replica --frontendIP 172.18.200.101 --listen 172.18.200.103:9502 --size 10G /stor2
+docker run -d --network="host" -P --expose 9502-9504 --expose 9700-9800 -v /mnt/stor2:/stor2 --name Replica-2 <openebs/jiva> launch Replica --frontendIP 172.18.200.101 --listen 172.18.200.103:9502 --size 10G /stor2
```
Comment (kmova) [inline README.md]: "launch Replica" -> "launch replica"

Comment (vharsh) [inline README.md]: `replica` again

**Comment by vharsh** [comment]
@AliceWonderland Create an issue in https://github.com/openebs/openebs/issues and reference it here.

**Comment by AliceWonderland** [comment]
Changed replica and controller!

**Comment by vharsh** [comment]
Fixes #openebs/501

**Comment by nitisuryawanshi** [comment]
@AliceWonderland Thank you so much for your PR contribution! We would like to mention you in our tweets, thanking you for your contribution. Can you please send me your twitter handle on this mail ID <niti.suryawanshi@openebs.io> ?

**Pull request merged**